Bugzilla/Report/SecurityRisk.pm

```diff
@@ -107,6 +115,34 @@ has 'results' => (
     ],
 );
 
+has 'deltas' => (
```

Comment:
the results are built from present -> past, but deltas must be made from past -> present. also deltas are only needed between the current week and last week. that's why we have a new attribute here instead of adding it to the results attribute